### PR TITLE
Added Content to readme file

### PR DESCRIPTION
**Pull request by josephgarner**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,68 +1,69 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Hack Zero
+> Hackathon management web application <br>use in tandem with [Hack-Zero-Backend](https://github.com/hackthehack/hack-zero-v2-backend)
 
-## Available Scripts
+[![NPM Version][npm-image]][npm-url]
+[![Build Status][travis-image]][travis-url]
```
Comment (jpeyper) [inline README.md:5]: were not using travis, you can get icons for github actions

File: README.md [hunk 2]
```diff
@@ -1,68 +1,69 @@
...
 
-In the project directory, you can run:
+Hack Zero is a serverless hackathon management system. It allows entities to create a single location to share
```
Comment (jpeyper) [inline README.md:7]: you probably want to define what an entity is

File: README.md [hunk 3]
```diff
@@ -1,68 +1,69 @@
...
+and update participates on the details and happenings of a hackathon event.
+Participates are also able to sign up to gain access to a range of features.
 
-### `npm start`
+### Key Features
+ * User Registration
+ * User Login
+ * Hack Idea Creation
+ * Join Hack Idea Team
+ * Editing Hacks
```
Comment (jpeyper) [inline README.md:16]: You might want to include a list of planned features (if we a defined one somewhere)

File: README.md [hunk 4]
```diff
@@ -1,68 +1,69 @@
...
+## Development setup
 
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+You will need to install Node.js in order to run this application, please refer to the installation above
+<br>
+The .env file will need to be updated with the correct API url:
+```sh
+REACT_APP_API_URL=[url]
+```
+the Contentful connection will also beed to be updated in the .env file:
```
Comment (jpeyper) [inline README.md:47]: Do they need to setup anything within their contentful space?

File: README.md (hunk 1 above)
Comment (josephgarner) [reply README.md:5]: ah yep!

File: README.md (hunk 3 above)
Comment (josephgarner) [reply README.md:15]: Added one!

File: README.md (hunk 4 above)
Comment (josephgarner) [reply README.md:52]: I'll double check with Stanley

**Pull request merged**